controllers/OrganisasiController.js

```
@@ -0,0 +1,34 @@
+        response(res, "Data retrieved successfully", organisasi, 200);
+    } catch (error) {
+        console.error(error);
+        response(res, error.message, null, 500);
+    }
+}
+
+export const createOrganisasi = async (req, res) => {
+    const { id, nama_organisasi, jabatan,  } = req.body;
+    try {
+        const organisasi = await Organisasi.create({
+            organisasi,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)